app/controllers/direct_assessments_controller.rb

```diff
@@ -2,6 +2,9 @@ class DirectAssessmentsController < ApplicationController
   def new
     @outcome = Outcome.find(params[:outcome_id])
     @assessment = DirectAssessment.new
```

Comment:
how about:

```
DirectAssessment.new(new_assessment_attributes)
```

Then the conditional on the next line can move into the attributes method and either return an empty hash or the attributes you want to clone. There's something about a conditional that makes me want to have it be isolated to an entire method rather than inside a method. Don't feel super strongly though


The difference is pretty minor, but I think I'd prefer the